php/unic/public/index.php

```diff
@@ -2,8 +2,10 @@
 require(__DIR__.'/../vendor/autoload.php');
 
 use Unic\App;
+use Swoole\Http\Server;
 
 $app = new App();
+$server = new Server('0.0.0.0', 3000);
```

Comment:
You let swoole use it's default values. Maybe you should dig into https://github.com/the-benchmarker/web-frameworks/blob/771608e476428245d58d0d9ef1c36583450c801f/php/swoole/start.php#L8

The most important is to disable log output